src/app/modules/core/advertisement/components/advertisement-list/sidebar/facility-filter/facility-filter.component.ts

```diff
@@ -1,11 +1,13 @@
-import { Component, OnInit } from '@angular/core'
+import { Component, EventEmitter, OnInit, Output } from '@angular/core'
 
 @Component({
   selector: 'cav-facility-filter',
   templateUrl: './facility-filter.component.html',
   styleUrls: ['./facility-filter.component.scss'],
 })
 export class FacilityFilterComponent implements OnInit {
+  @Output() filter = new EventEmitter<object>()
```

Comment:
Invece di instaziare un `EventEmitter<object>`, potresti creare un'interfaccia che abbia come proprietà `type: string, active:boolean`, così da avere un controllo in più, sia quando emitti questo filter, sia quando prendi i parametri nell'html, perché dovranno rispettare le proprietà descritte nell'interfaccia